version.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/main/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "2.0-preview",
+  "version": "1.7-preview",
```

Comment:
Should we bump the version?

Reply:
The latest released version is 1.4. As per extension [versioning stragegy](https://github.com/dbos-inc/ttdbg-extension?tab=readme-ov-file#versioning-strategy) this should *NOT* have been set to an even number. Our next release will be 2.0, so this changes the extension version to an odd value lower than 2.0 that we haven't used yet